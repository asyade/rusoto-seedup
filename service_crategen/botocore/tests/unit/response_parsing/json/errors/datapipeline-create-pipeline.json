{"__type":"AccessDeniedException","Message":"Operation createPipeline is not allowed to be performed with role identity"}
