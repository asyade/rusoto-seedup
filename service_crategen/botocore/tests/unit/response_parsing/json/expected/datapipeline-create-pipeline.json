{
    "Error": {
        "Message": "Operation createPipeline is not allowed to be performed with role identity",
        "Code": "AccessDeniedException"
    },
    "ResponseMetadata": {}
}
